PowerShell/ScubaGear/Testing/Unit/PowerShell/Orchestrator/Invoke-RunCached.Tests.ps1

```diff
@@ -16,6 +16,8 @@ InModuleScope Orchestrator {
             Mock -ModuleName Orchestrator Invoke-RunRego {}
             function Invoke-ReportCreation {}
             Mock -ModuleName Orchestrator Invoke-ReportCreation {}
+            function Merge-JsonOutput {}
+            Mock -ModuleName Orchestrator Merge-JsonOutput {}
```

Comment:
Not required - helps make sure we are using the mocked version and not the what I think of as a stubbed definition. 

```suggestion
            function Merge-JsonOutput {throw 'this will be mocked'}
            Mock -ModuleName Orchestrator Merge-JsonOutput {}
```